zc_install/sql/updates/mysql_upgrade_zencart_157.sql

```diff
@@ -35,6 +35,65 @@
 TRUNCATE TABLE whos_online;
 TRUNCATE TABLE db_cache;
 
+# Remove music
+DROP TABLE IF EXISTS `media_clips`;
```

Comment:
WOW!!!! So, going to just wipe everyone's listings and data??? Isn't this up to the store owner/user? Is this encouragement ***NOT*** to upgrade?

Adding an option to prevent this removal???

Reply:
In ten years of doing this, I know exactly ZERO people who are using this stuff.  The small handful of people who actually do use it can tweak this upgrade to not do the removal; it's all in one place and can be removed.    I don't want to lug this stuff around anymore; I want to make room for new useful capabilities that will actually benefit the majority of stores.

Reply:
Hold your breath ... I agree with @mc12345678 on this!  It's one thing to not include the tables on an initial install, but to remove them on an upgrade?

Since the majority (I'm just guessing) of Zen Cart store-owners follow the forums more closely than the GitHub, how about posting something on the forums that indicates that the removal is "under consideration" to see how much blow-back is received?